firmware/components/driver_framebuffer/include/driver_framebuffer_compositor.h

```diff
@@ -31,8 +32,13 @@ typedef struct Window_t {
 	
 	/* Buffer */
 	uint8_t* buffer;
+
+	/* Graphics acceleration */

```

Comment:
There's no hardware acceleration, so probably best to call this something like "graphics transformations"

Reply:
True, might be wise to change this indeed